docs/migrating-configs.md

```diff
@@ -11,6 +11,44 @@ Occasionally, there are changes made to Ignition's configuration that break back
 1. TOC
 {:toc}
 
+## From Version 3.2.0 to 3.3.0
+
+There are not any breaking changes between versions 3.2.0 and 3.3.0 of the configuration specification. Any valid 3.2.0 configuration can be updated to a 3.3.0 configuration by simply changing the version string in the config.
+
+The following is a list of notable new features, deprecations, and changes.
+
+### Kernel arguments
+
+Ignition now supports specifying kernel arguments as part of the Ignition config. For example:
+
+<!-- ignition -->
+```json
+{
+  "ignition": { "version": "3.3.0" },
+  "kernelArguments": {
+    "shouldExit": [
+      "foobar"
+    ],
+    "shouldNotExist": [
+      "raboof"
+    ]
+  }
+}
+```
+
+The config above will ensure that the kernel argument `foobar` is present, and the kernel argument `raboof` is absent.
+
+This feature requires integration with the distribution where Ignition is used. See the [distributor notes][distributor-notes] for details.
+
+### Clevis API changes
+
+The Golang API related to Clevis has changed:
+- The `Clevis` member of the `Luks` type is no longer a pointer to a struct, but a struct.
+- The `Custom` type has been renamed `ClevisCustom`.
+- The `Custom` member of the `Clevis` type is no longer a pointer to a struct, but a struct.
```

Comment:
There are also the pointerification changes.  But historically we haven't documented Go API changes in similar circumstances, since this is a user-facing document.  (And the compiler will catch any use of the old APIs.)